app/controllers/vendor_api/vendor_api_controller.rb

```diff
@@ -1,4 +1,11 @@
 module VendorApi
   class VendorApiController < ActionController::API
+    before_action :set_cors_headers
+
+  private
+
+    def set_cors_headers
+      headers['Access-Control-Allow-Origin'] = '*'
```

Comment:
If at some point we introduce cookie authentication and don't make absolutely sure that API access is _only_ granted to users via token auth, then if I'm logged in to the service with a cookie then `$malicious_website` could cause me to issue authenticated API requests. Could we scope it to Swagger editor, maybe?

Reply:
This header only applies to the Vendor API, I don't think we'll add cookie auth for that?